schema.sql

```diff
@@ -0,0 +1,70 @@
+CREATE TABLE book (
+    id int GENERATED BY DEFAULT AS IDENTITY,
+    genre_id int,
+    author_id int,
+    source_-id int,
+    label_id int,
+    publish_date date,
+    archived boolean,
+    publisher varchar(50),
+    cover_state varchar(50),
+    PRIMARY KEY (id),
```

Comment:
- [optional] You may not necessarily need line 11 (PRIMARY KEY (id)), line 1 (    id int GENERATED BY DEFAULT AS IDENTITY,) is typically used for creating a primary key column for the table. **Note: This is applicable to all the tables.**

Reply:
I will work on that now.